css/index.css

```diff
@@ -137,10 +297,100 @@ footer nav {
     justify-content: space-between;
     align-items: center;
     padding: 20px 2%;
-    font-size: 14px;
+    font-size: 1.4rem;
 }
 
 footer nav a {
     color: white;
```

Comment:
Awesome job Charletta with your breakpoints.  They work just as needed and required.  Your design looks great as well.